Makefile

```diff
@@ -98,7 +98,8 @@ test_unit: clean static_external
 
 # Check whether migrations need to be generated
 test_migrations_missing: clean
-	! honcho -e .env.test run ./manage.py makemigrations --dry-run --exit
+	(psql -lqt | cut -d \| -f 1 | grep -qw opencraft) || (createdb --encoding utf-8 --template template0 opencraft)
```

Comment:
@antoviaque Yup, that's what it does. As for why it's necessary, cf. ac9be4f9b22e93c1bcd3278eff8c270bbe1dc669: After the updates, `makemigrations` fails with "OperationalError: database ... does not exist" if the target database does not exist (this is actually what I would expect). I first noticed this on CircleCI, and then also verified it locally, by changing `DATABASE_URL` in `.env.test` to point to a database that didn't exist on my Vagrant box and running `make test_migrations_missing`. Didn't get any errors with the old dependencies/master, and saw the error mentioned above after switching back to the new dependencies/this branch.